sdk/core/azure-core/test/ut/uuid.cpp

```diff
@@ -13,3 +13,17 @@ TEST(Uuid, Basic)
   auto uuid = Uuid::CreateUuid();
   EXPECT_TRUE(uuid.GetUuidString().length() == 36);
 }
+
+TEST(Uuid, Randomness) 
+{ 
+  const int Size = 100000;
+  std::set<std::string> uuids;
+  for (int i = 0; i < Size; i++)
+  {
+    auto ret = uuids.insert(Uuid::CreateUuid().GetUuidString());
+    //If the value already exists in the set then the insert will fail
+    //  ret.second == false means the insert failed.
```

Comment:
Did you intend for the second space on line 25 to be on line 24?
```suggestion
    // If the value already exists in the set then the insert will fail
    // ret.second == false means the insert failed.
```

Reply:
Fixed